DEV-GUIDE.md

```diff
@@ -80,6 +89,11 @@ export GOPATH=~/go
 export PATH=$PATH:$GOPATH/bin
 export ISTIO=$GOPATH/src/istio.io # eg. ~/go/src/istio.io
 
+# Please change HUB and TAG to a desired HUB and TAG for custom docker
+# container builds.
+export HUB="gcr.io/yourid"
+export TAG=`whoami`
```

Comment:
Isn't there a default in the makefile to make a tag with username and date ?

Reply:
Ya.  I'll remove the tag or give greater explanation as to when TAG should be overridden.